### ISSUE TITLE
Build farmer handoff 2017-11-01

### ISSUE THREAD

**Issue by nuclearsandwich** [ros2/build_farmer#65]
## Buildfarm health

There was a day of delayed nightlies and agent connectivity issues due to the wiline outage 29 October.

### Tasks in progress or withheld

I'm going to keep these in my backlog, using GitHub issue assignment to signify such.

- apt update on Ubuntu hosts (hopefully solving #30)
- Investigate packet.net host issue (#42)
- Upgrade eatable to Windows 10 (#17) (Sched. 2017-11-17)

#### Tasks pending

- [x] Update Java on Windows (#43)

#### Nice-to-haves

- Try VS2017 and assess if we can migrate Windows nodes to it
  - BLOCKED on Connext https://github.com/ros2/build_cop/issues/61#issuecomment-337955537 Is it worth ticketing in the meantime so we can drop this "nice to have" thing?

#### Recurring tasks

- system and package (brew/chocolatey) updates on MacOS and Windows machines
  - [ ] dosa (last updated 2017-09-13)
  - [ ] mini1 (last updated 2017-09-13)
  - [ ] mini2 (last updated 2017-09-13)
  - [ ] portable (last updated 2017-09-13)
  - [ ] windshield (last updated 2017-09-13)

## Test suite health

Nothing out of the ordinary beyond https://github.com/ros2/build_cop/issues/64, which is resolved. Did not try to diminish flaking tests this pass as I was trying to focus more on buildfarm health.

**Comment by clalancette** [comment]
Thanks!

**Comment by nuclearsandwich** [comment]
I meant to do this before the handoff and only just remembered to do it now.

I've added some labels to this repo to try and better track activity.

`chore`: one and done tasks where the issue can be closed when the task is performed.
`buildfarm`: issues related to the build / infrastructure itself. Jenkins and the stuff outside it.
`test-suite`: issues related to instrumenting the test suite from within jenkins. Issues with specific tests should be opened on the source issue tracker.

I'm not sure these will be helpful to everyone but knowing how much time we spend on infrastructure will help us prioritize bigger scope infra work.

**Comment by clalancette** [comment]
Next handoff is in #70; closing this one out.
